src/layouts/dialog.vue

```diff
@@ -16,33 +16,29 @@ export default Vue.extend({
     // the "dark" part will be a dynamic pref.
     // See https://github.com/rancher/dashboard/blob/3454590ff6a825f7e739356069576fbae4afaebc/layouts/default.vue#L227 for an example
     return { bodyAttrs: { class: 'theme-dark' } };
-  },
-  mounted() {
-    const wrapper = this.$refs.wrapper as HTMLDialogElement;
-
-    // Dynamically resize the window to fit the contents.
-    // We need a bit extra on the width to ensure we don't get scroll bars.
-    window.resizeBy(
-      wrapper.offsetWidth - document.documentElement.offsetWidth + 14,
-      wrapper.offsetHeight - document.documentElement.offsetHeight);
```

Comment:
I didn't see this really doing anything to our window.. Perhaps something changed since the original implementation?